docs/code-snippets/excel-snippets.yaml

```diff
@@ -2316,6 +2316,17 @@ Excel.Worksheet#onDeactivated:member:
         });
         return context.sync();
     });
+Excel.Worksheet#onRowHiddenChanged:member:
+  - |-
+    Excel.run(function (context) {
+        const sheet = context.workbook.worksheets.getActiveWorksheet();
+        sheet.onRowHiddenChanged.add(function (event)   {
+        return Excel.run(async (context) => {
```

Comment:
nit: Should these 2 lines be indented further (this line and the one following)?

Reply:
Yes. Also, I should remove the `async` and `=>` from that return line. Some of the other snippets have that problem too. Rerunning GenerateDocs...